src/plugins/event-plugins/__integ__/WebVitalsPlugin.test.ts

```diff
@@ -61,3 +62,58 @@ test('WebVitalEvent records lcp and cls events on chrome', async (t: TestControl
         .expect(clsEventDetails.attribution)
         .typeOf('object');
 });
+
+test('when lcp image resource is recorded then it is attributed to lcp', async (t: TestController) => {
```

Comment:
I believe this test is correct but there is something wrong with either TestCafe or ResourcePlugin that is causing this to fail.